shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -81,6 +81,19 @@ static bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
   return transformed_rrect.contains(platformview_boundingrect);
 }
 
+// Determines if the intersection rect of a layer and a platform view is a single pixel width
+// or height, and on the edge of the platform view.
+static bool IsIntersectionSinglePixelWidthOrHeightOnEdgeOfPlatformView(
```

Comment:
This entire function is equivalent to: `!intersection_rect.intersects(roundedInPVRect)`

Reply:
Smart!

Reply:
Updated. This made my day!